tests/in/atomicSimple.wgsl

```diff
@@ -0,0 +1,49 @@
+var<workgroup> foo: atomic<u32>;
+var<workgroup> baz: array<atomic<u32>, 2>;
+
+@compute
+@workgroup_size(2)
+fn corge(@builtin(local_invocation_id) id: vec3<u32>) {
```

Comment:
nitpick: `corge` is random gibberish that served its purpose while I was still spending brain cycles understanding other things. Maybe something more sensible (even if bland, like `cs_main`) should be used here?

Reply:
Renamed to `cs_main`.